cmdline-pull/pubsub_sample.py

```diff
@@ -104,80 +74,73 @@ def list_topics(client, args):
 
 
 def list_subscriptions(client, args):
-    """Shows the list of current subscriptions."""
-    next_page_token = None
-    while True:
-        resp = client.projects().subscriptions().list(
-            project='projects/{}'.format(args[0]),
-            pageToken=next_page_token).execute(num_retries=NUM_RETRIES)
-        for subscription in resp['subscriptions']:
-            print json.dumps(subscription, indent=1)
-        next_page_token = resp.get('nextPageToken')
-        if not next_page_token:
-            break
+    """Show the list of current subscriptions attached to a given topic.
 
-
-def list_subscriptions_in_topic(client, args):
-    """Shows the list of subscriptions attached to a given topic."""
+    If no topic is specified, all current subscriptions associated with a
+    project will be listed."""
```

Comment:
Can you add a newline after the docstring like:

```python
    def list_subscriptions_in_topic(client, args):
        """Show the list of current subscriptions.

        If a topic is given, list subscriptions associated with the topic, otherwise
        list all the subscriptions in the project.
        """
```